### ISSUE TITLE
Question about the initialization of sampling_offsets

### ISSUE THREAD

**Issue by Mayy1994** [sail-sg/mvp#21]
Dear authors,

In lib.models/ops/modules/projattn.py, I noticed that the weights of self.sampling_offsets is set to constant 0, and the bias has no gradient backpropagation (line 94- line 105).

https://github.com/sail-sg/mvp/blob/80eecd012f51f49da357e337716d40a6398d520d/lib/models/ops/modules/projattn.py#L94
<img width="597" alt="屏幕快照 2022-07-19 下午3 51 07" src="https://user-images.githubusercontent.com/40232175/179699109-ba847f6a-fccd-468b-93bd-10220883636c.png">

In my opinion, if the weights are set to 0 and the bias has no gradient, the sampled offsets will be always the same across different training samples. But it seems the offsetted points are informatively selected according to Figure 5 in your paper.

On the other hand, in the provided pretrained model, the weights and the bias are different from what they are initialized.
Could you please tell me what is the final initialization method of self.sampling_offsets? 
Thank you very much!
